api/src/main/java/org/openmrs/module/aijar/api/AijarService.java

```diff
@@ -38,4 +42,41 @@
 	public void linkExposedInfantToMotherViaARTNumber(Patient infant, String motherARTNumber);
 	public void linkExposedInfantToMotherViaARTNumber(Person infant, String motherARTNumber);
 	public void setAlertForAllUsers(String alertMessage);
+
+	/**
+	 * Transfer out Information for patient
+	 * @param patient
+	 * @return Map
+	 */
+	public Map transferredOut(Patient patient);
+
+
+	/**
+	 * Transfer In Information for patient
+	 * @param patient
+	 * @return Map
+	 */
+	public Map transferredIn(Patient patient);
+
+	/**
+	 * Check if Patient is transferred out. This method depends on transferredOut(Patient patient) method
+	 * @param patient
+	 * @return boolean
+	 */
+	public boolean isTransferredOut(Patient patient);
```

Comment:
@slubwama This method needs a start date and an end date to ascertain period during which the check was made, important for reports